wp-content/plugins/tw-calls-to-action/tw-calls-to-action.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/*
+Plugin Name: TW Calls to Action
+Description: Defines the Sidebars that house Call to Actions
+*/
+
+/**
+ * Register widget area.
+ *
+ * @link https://developer.wordpress.org/themes/functionality/sidebars/#registering-a-sidebar
+ */
+function tw_calls_to_action_widgets_init() {
+
+	register_sidebar(
```

Comment:
These sidebars were originally housed in functions.php in the newspack theme. 

We may want to try to create our own regions.